input-output/pydicom_series.py

```diff
@@ -504,7 +504,7 @@ def description(self):
 
         # Give patient name
         if 'PatientName' in info:
-            fields.append("" + info.PatientName)
```

Comment:
this construct with an empty string is a bit strange, probably just use `fields.append(f"{info.PatientName})`

Reply:
True, no need for that. Replaced it with the f-string now.